main.go

```diff
@@ -116,6 +117,12 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "Glance")
 		os.Exit(1)
 	}
+	if strings.ToLower(os.Getenv("ENABLE_WEBHOOKS")) != "false" {
```

Comment:
This can be made more user friendly with  https://pkg.go.dev/strconv#ParseBool

Reply:
@gibizer we have prior art for this in other operators. Lets follow up on stuff like this later I think